### PR TITLE
Volume Health uses a controller and kubelet only

### LINKED ISSUE #544: Volume Health Monitoring Feature docs out of date
In the [CSI driver docs](https://kubernetes-csi.github.io/docs/volume-health-monitor.html) it says that:

> The [External Health Monitor](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1432-volume-health-monitor) is implemented as two components: External Health Monitor Controller and External Health Monitor Agent.

However the [external-health-monitor readme](https://github.com/kubernetes-csi/external-health-monitor/blob/master/README.md) says that:

> The [Volume Health Monitor](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1432-volume-health-monitor) is implemented in two components: External Health Monitor Controller and Kubelet.

It looks like https://github.com/kubernetes-csi/external-health-monitor/blob/master/README.md is up to date but the docs aren't

### PR DESCRIPTION
**Pull request by coulof**
Remove references to deprecated module.

Mostly a copy-paste from https://github.com/kubernetes-csi/external-health-monitor/blob/v0.8.0/README.md

Fixes #544

### CODE REVIEW

File: book/src/volume-health-monitor.md [hunk 1]
```diff
@@ -2,37 +2,32 @@
 
 ## Status
 
-Status | CSI external-health-monitor-controller sidecar Version | CSI external-health-monitor-agent sidecar Version
---|--|--
-Alpha | 0.1.0 | 0.1.0
+Status | Min K8s Version | Max K8s Version | external-health-monitor-controller Version
+--|--|--|--
+Alpha | 1.21 | - | 0.3.0
```
Comment (xing-yang) [inline book/src/volume-health-monitor.md:7]: Can you list the latest external-health-monitor-controller Version which is 0.8.0?

Comment (coulof) [reply book/src/volume-health-monitor.md]: Done

**Pull request merged**